app/models/models.py

```diff
@@ -1,24 +1,89 @@
 """import datetime to help you get the current date and time."""
 from datetime import datetime
 from werkzeug.security import generate_password_hash
-parties = []
-offices = []
-users = []
+import psycopg2
+from flask import current_app
 
 
-class Parties:
+
+class DatabaseConnection:
+    def __init__(self):
+        self.host = current_app.config["DB_HOST"]
+        self.name = current_app.config["DB_NAME"]
+        self.username = current_app.config["DB_USERNAME"]
+        self.password = current_app.config["DB_PASSWORD"]
+
+        self.conn = psycopg2.connect(
+            host=self.host,
+            database= self.name,
+            password=self.password,
+            user =self.username
+            )
+
+
+        self.cursor = self.conn.cursor()
+
+
+
+class Parties(DatabaseConnection):
     """ create class party that will hold party related data."""
-    party_id = 1
 
     def __init__(self, name=None, hqAddress=None, logoUrl=None):
+        
+        super().__init__()
         self.name = name
         self.hqAddress = hqAddress
         self.logoUrl = logoUrl
-        self.id = Parties.party_id
         self.date_created = str(
             datetime.now().replace(microsecond=0, second=0))
+        self.id = None
+
+
+    def create_tables(self):
+
+        self.cursor.execute(
+        """
+        CREATE TABLE parties (
+            id serial PRIMARY KEY,
+            name VARCHAR NOT NULL,
+            hqAddress VARCHAR NOT NULL,
+            logoUrl VARCHAR NOT NULL,
+            date_created TIMESTAMP
+        );
+        """
+        )
+        self.conn.commit()
+
+
+    """ function to drop table parties """
```

Comment:
This comment is wrongly placed. I think I shared some resources with you on how to use `docstrings`. Please check it out if you forgot and correct all every comment which is wrongly placed.